structured/src/main/java/com/softwaremill/jox/structured/Race.java

```diff
@@ -146,11 +143,11 @@ public static <T> T raceResult(List<Callable<T>> fs) throws InterruptedException
         }
     }
 
-    private record NullWrapperInRace() {}
-
     private record ExceptionWrapperInRace(Exception e) {}
 
     private record ExceptionWrapperInRaceResult(Exception e) {}
-
-    private record Timeout() {}
+}
+enum MagicConstants {
```

Comment:
👍 For consistency, can this be moved as `private` inside the `Race` class? It has a similar role as `ExceptionWrapperInRace` etc